api/MediaDevices.json

```diff
@@ -115,6 +115,72 @@
           }
         }
       },
+      "devicechange_event": {
+        "__compat": {
+          "description": "<code>devicechange</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDevices/devicechange_event",
+          "support": {
+            "chrome": {
+              "version_added": "57"
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": "52"
+              },
+              {
+                "version_added": "51",
+                "version_removed": "52",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "media.ondevicechange.enabled",
```

Comment:
Assume this pref covers the event as well as the property. Although tbh you could just remove this clause and say support starts at 52.